src/celeritas/em/params/WentzelOKVIParams.cc

```diff
@@ -0,0 +1,929 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/params/WentzelOKVIParams.cc
+//---------------------------------------------------------------------------//
+#include "WentzelOKVIParams.hh"
+
+#include <algorithm>
+#include <utility>
+
+#include "corecel/io/Logger.hh"
+#include "corecel/sys/ScopedMem.hh"
+#include "celeritas/io/ImportData.hh"
+#include "celeritas/mat/MaterialParams.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct if Wentzel VI or Coulomb is present, else return nullptr.
+ */
+std::shared_ptr<WentzelOKVIParams>
+WentzelOKVIParams::from_import(ImportData const& data,
+                               SPConstMaterials materials)
+{
+    CELER_EXPECT(materials);
+
+    bool wentzel = has_msc_model(data, ImportModelClass::wentzel_vi_uni);
+    bool coulomb = has_model(data, ImportModelClass::e_coulomb_scattering);
+    if (!(wentzel || coulomb))
+    {
+        // No Wentzel VI MSC or Coulomb scattering present
+        return nullptr;
+    }
+
+    // Use combined single and multiple Coulomb scattering if both the single
+    // scattering and the Wentzel VI models are present
```

Comment:
Again, this description is somewhat confusing as it sounds like that two models are combined together.  The actual combination is to implement for cases for the single scattering (not the Coulomb scattering model itself) and the WentzelVI multiple scattering using the common WentzelOKVI cross section calculation, which are I think technically different.